src/lib/rest/httpRequestSend.cpp

```diff
@@ -266,8 +267,21 @@ int httpRequestSendWithCurl
   int                             outgoingMsgSize       = 0;
   std::string                     content_type(orig_content_type);
   std::map<std::string, bool>     usedExtraHeaders;
+  char                            servicePath0[64];  // 64 > 50 (max component length in service path)
```

Comment:
Including slashesh which separate components?

By the way, we should have the max for SP string in limits.h... Otherwise, it should be added.